README.md

```
@@ -1,92 +1,120 @@
 
-[Better Auth TanStack](https://github.com/daveyplate/better-auth-tanstack)
+- Node.js (LTS version)
+- pnpm 10.6.5 or later
+- PostgreSQL database
 
-[TanStack Query](https://tanstack.com/query)
+### Installation
 
-[shadcn/ui](https://ui.shadcn.com)
+1. Clone the repository:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)